src/ecrc-frontend/src/components/page/applicationForm/ApplicationForm.js

```diff
@@ -213,7 +241,56 @@ export default function ApplicationForm({
     type: "submit"
   };
 
-  const applicationVerification = () => {};
+  const applicationVerification = () => {
+    if (birthPlace === "") {
+      setBirthPlaceError("Please enter your city and country of birth");
+    }
+
+    if (phoneNumber === "") {
+      setPhoneNumberError("Please enter your primary phone number");
+    }
+
+    if (emailAddress === "") {
+      setEmailAddressError("Please enter your personal email address");
+    }
+
+    if (jobTitle === "") {
+      setJobTitleError("Please enter your position/job title");
+    }
+
+    if (defaultScheduleTypeCd === "WBSD" && organizationFacility === "") {
+      setOrganizationFacilityError("Please enter your organization facility");
+    }
+
+    if (
+      birthPlace !== "" &&
+      phoneNumber !== "" &&
+      emailAddress !== "" &&
+      jobTitle !== "" &&
+      !(defaultScheduleTypeCd === "WBSD" && organizationFacility === "")
+    ) {
+      setApplicant({
+        firstName,
+        middleName,
+        lastName,
+        birthDate,
+        sex,
+        street,
+        city,
+        province,
+        postalCode,
+        country,
+        birthPlace,
+        driversLicNo,
+        phoneNumber,
+        emailAddress,
+        jobTitle,
+        organizationFacility
+      });
+
+      history.push("/");
```

Comment:
question: I know we are modifying history here and many other places, but should we be using 
`Redirect />` instead?

Reply:
Mabey? I'm not 100% on what the options are for react router, so just coppied how you had used it the first time, can put some time looking into this tomorrow.

Reply:
yeah I was reading into it a bit, and I think using Redirect actually indirectly modifies the history and is recommended to be used rather than modifying it directly